### PR TITLE
Update foreman_remote_execution to 1.8.4

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: plugins/ruby-foreman-remote-execution/debian/install [hunk 1]
```diff
@@ -1,5 +1,5 @@
 foreman_remote_execution.rb           usr/share/foreman/bundler.d
 cache                                 usr/share/foreman/vendor
 assets                                var/lib/foreman/public
-webpack                               var/lib/foreman/public
+webpack/foreman_remote_execution      var/lib/foreman/public/webpack
```
Comment (adamruzicka) [inline plugins/ruby-foreman-remote-execution/debian/install]: Interesting, I wasn't aware I'd have to change anything else than the versions

Comment (mmoll) [reply plugins/ruby-foreman-remote-execution/debian/install]: See GH-4025 for the full story, I just wanted to take the opportunity to bring this in with the udpate.

Comment (adamruzicka) [reply plugins/ruby-foreman-remote-execution/debian/install]: Ok, makes sense. Thanks for stepping in

**Comment by mmoll** [comment]
merged, díky @adamruzicka!

**Pull request merged**